pkg/beacon/relay/entry/states.go

```diff
@@ -136,12 +142,31 @@ func (scs *signatureCompleteState) Initiate(ctx context.Context) error {
 				err,
```

Comment:
We should also refactor this case: if a share could not be unmarshalled or is not valid, we log a warning and continue. We currently reject invalid `bn256.G1` shares but still let the malicious member break the entire protocol by broadcasting invalid public key share `[]byte`. This can lead to an attack when a member breaks signing for the entire group and reports entry timeout to get a tattletale reward.

Reply:
Ah, we are not returning here, my bad! Let's just change it to `Warningf`.